examples/basic/one_broker/inventory_vm.yml

```diff
@@ -0,0 +1,27 @@
+#
+# This sample inventory file defines a node that provides a Broker component
+# which is running in the local machine (ansible_connection: docker).
+#
+# The following node variables are mandatory:
+# - component (determines a component that can be parsed by iqa [router, broker, client])
+# - implementation (for broker, we currently just have: artemis)
+#
+# Broker component required variables:
+# - broker_name
+# - broker_web_port
+# - broker_path
+# - service (not needed when running as a container)
+# - executor (defines the executor instance to be used for communicating with the node)
+#
+all:
+  children:
+    brokers:
+      hosts:
+        broker1:
+          ansible_host: <host-ip>
```

Comment:
@michalxo Instead of using a string to be replaced, please use something that can be resolved, in case someone tries it. Like using an env var or default to '127.0.0.1'.

Reply:
Updated, added 127.0.0.1 there (force pushed commit)